src/modules/account/containers/transactions-box.ts

```diff
@@ -0,0 +1,43 @@
+import { connect } from "react-redux";
+import getRep from "modules/account/actions/get-rep";
+import { updateModal } from "modules/modal/actions/update-modal";
+import { closeModal } from "modules/modal/actions/close-modal";
+import { MODAL_REVIEW } from "modules/common-elements/constants";
+import { transferFunds } from "modules/auth/actions/transfer-funds";
+import { selectLoginAccount } from "modules/auth/selectors/login-account";
+import { getGasPrice } from "modules/auth/selectors/get-gas-price";
+import { TransactionsBox } from "modules/account/components/transactions/transactions-box";
+// made state an ANY for now.
+const mapStateToProps = (state: any) => {
+  const loginAccount = selectLoginAccount(state);
+  
+  return {
+    augurNodeNetworkId: state.connection.augurNodeNetworkId,
```

Comment:
just go ahead and convert `state.connection.augurNodeNetworkId,` to IS_MAINNET or allowREPFaucet or something so we don't have `1` in the component.